pair_test.rb

```diff
@@ -1,5 +1,6 @@
 class Test
   def initialize(test)
     @test = test
+    @add_content = nil
```

Comment:
Looks awesome!

Reply:
Totes, it's just nice to see SOMETHING work as planned :]